DigitalLearningSolutions.Data/Helpers/ConfigHelper.cs

```diff
@@ -8,6 +8,7 @@ public static class ConfigHelper
         public const string CurrentSystemBaseUrlName = "CurrentSystemBaseUrl";
         private const string LearningHubOpenApiKey = "LearningHubOpenAPIKey";
         private const string LearningHubOpenApiBaseUrl = "LearningHubOpenAPIBaseUrl";
+        public const string UseSignposting = "FeatureManagement:UseSignposting";

```

Comment:
Does this one need to be public? I don't see why we'd use this outside the helper?

(I'd ask the same about the CurrentSystemBaseUrlName but that isn't in scope for this)

Reply:
Right, changed to private